markdown/developers/tutorials/dsl2_modules_tutorial.md

```diff
@@ -384,13 +471,13 @@ In order to carry out the test, _pytest-workflow_ will search for information st
 
```

Comment:
`FGBIO_FASTQTOBAM ( input )` has not been changed to ` FGBIO_DEMOFASTQTOBAM ( input )`